changelog.txt

```diff
@@ -1,3 +1,21 @@
+4.36.0
+==========
+### Bug Fixes:
```

Comment:
Do we have the Bug fixes section before New features? this wasn't the case before 4.32.0?